sdk/batch/Microsoft.Azure.Batch/tests/ProtocolTests/IntegrationJobApplicationPackageReferencesTests.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿namespace BatchProxyIntegrationTests
+{
+    using System;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using BatchTestCommon;
+    using IntegrationTestCommon;
+    using Microsoft.Azure.Batch.Protocol;
+    using Microsoft.Azure.Batch.Protocol.Models;
+    using Microsoft.Azure.Management.Batch;
+    using Microsoft.Rest.Azure;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class IntegrationJobApplicationPackageReferencesTests : IDisposable
+    {
+        private readonly string AccountName = TestCommon.Configuration.BatchAccountName;
+        private readonly string AccountKey = TestCommon.Configuration.BatchAccountKey;
+        private readonly string Url = TestCommon.Configuration.BatchAccountUrl;
+
+        private readonly ITestOutputHelper output;
+
+        private const string AppPackageName = "job-application-package-references-tests";
+        private const string Version = "1.0";
+
+        public IntegrationJobApplicationPackageReferencesTests(ITestOutputHelper output)
+        {
+            this.output = output;
+
+            IntegrationTestCommon.EnableAutoStorageAsync().Wait();
+
+            ApplicationPackageCommon.UploadTestApplicationPackageIfNotAlreadyUploadedAsync(AppPackageName, Version).Wait();
+            ApplicationPackageCommon.UpdateApplicationPackageAsync(AppPackageName, Version, "My First App", false).Wait();
+        }
+
+        //[Fact]
+        //[Trait(TestTraits.Duration.TraitName, TestTraits.Duration.Values.MediumDuration)]
```

Comment:
Was this stuff commented out before you moved it?

Reply:
Looks like it was

Reply:
No, it wasn't commented out earlier.
I did comment it out. This line
`var client = new BatchServiceClient(new Uri(Url), new BatchSharedKeyCredential(AccountName, AccountKey))` was an error due to wrong arguements.

>Error	CS1503	Argument 1: cannot convert from 'System.Uri' to 'Microsoft.Rest.ServiceClientCredentials'	Microsoft.Azure.Batch.Protocol.Tests (net461), Microsoft.Azure.Batch.Protocol.Tests (netcoreapp2.1)	

> Error	CS1503	Argument 2: cannot convert from 'Microsoft.Azure.Batch.Protocol.BatchSharedKeyCredential' to 'System.Net.Http.DelegatingHandler'	Microsoft.Azure.Batch.Protocol.Tests (net461), Microsoft.Azure.Batch.Protocol.Tests (netcoreapp2.1)